entity_test.go

```diff
@@ -2,8 +2,8 @@ package configkit_test
 
 import (
 	. "github.com/dogmatiq/configkit"
-	"github.com/dogmatiq/configkit/fixtures" // can't dot-import due to conflicts
-	"github.com/dogmatiq/configkit/message"
+	cfixtures "github.com/dogmatiq/configkit/fixtures" // can't dot-import due to conflicts
```

Comment:
Does this need to mention dot-import if its aliasing it anyway?

Reply:
~Yes, as it's only aliased because it conflicts with another `fixtures` package that also can't be dot-imported.~

Turns out the other fixtures package is no longer being imported anyway, so I've removed the alias.